chapters/connectors.tex

```diff
@@ -871,7 +871,7 @@ \subsection{Overconstrained Equation Operators for Connection Graphs}\label{over
 Connections.root(A.R)
 \end{lstlisting}\end{synopsis}
 \begin{semantics}
-The overdetermined type or record instance \lstinline!R! in connector instance \lstinline!A! is a (definite) \firstuse{root node}\index{definite root node}\index{root node!definite} in a virtual connection graph.
+The overdetermined type or record instance \lstinline!R! in connector instance \lstinline!A! is a (definite) \firstuse[definite root node]{root node}\index{root node!definite} in a virtual connection graph.
```

Comment:
May "(definite)" should be inside firstuse, as the index entry is definite root node.

Reply:
Sure, why not?  Done.  OK?